### DISCUSSION TITLE
Use suneditor to edit post

### DISCUSSION THREAD

**Discussion by jml-french** [danpros/htmly#846] [Ideas]
The markdown editor (and markdown in general) is really limited.
The WYSIWYG SunEditor (https://github.com/JiHong88/SunEditor) is a really powerful editing tool that allows for rich configuration and easy translations. It is also very well documented and maintained.
I have easily set it up on an application written in sqlpage (https://github.com/sqlpage/sqlpage/)
It would be interesting to include it as a replacement or an alternative tool.

**Comment by KuJoe** [comment]
SunEditor looks like an HTML editor, HTMLy uses Markdown for the pages (all pages are stored as .md files and not .html) so this wouldn't be compatible.
